src/components/MovieCatalog/MovieCard/index.jsx

```diff
@@ -1,32 +1,34 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
 
 import './style.scss';
 import MovieCardMenu from '../MovieCardMenu';
+import { setCurrentMovieId } from '../../../slices/moviesInfo';
 
-const MovieCard = ({ movie, genres }) => {
+const MovieCard = ({ movie }) => {
   const [visible, setVisible] = useState(false);
-
-  const handleClickOpenMenu = () => setVisible(true);
-  const handleClickCloseMenu = () => setVisible(false);
+  const dispatch = useDispatch();
```

Comment:
Do not forget to move all container components to the containers folder

Reply:
done